src/database/models/permissions.js

```diff
@@ -0,0 +1,40 @@
+module.exports = (sequelize, DataTypes) => {
+	const Permission = sequelize.define(
+		'Permission',
+		{
+			id: {
+				allowNull: false,
+				autoIncrement: true,
+				primaryKey: true,
+				type: DataTypes.INTEGER,
+			},
+			code: {
+				type: DataTypes.STRING,
+				allowNull: false,
+			},
+			module: {
+				type: DataTypes.STRING,
+				allowNull: false,
+			},
+			actions: {
+				allowNull: false,
+				type: DataTypes.ENUM('ALL', 'READ', 'WRITE', 'UPDATE', 'DELETE'),
+			},
+			status: {
+				type: DataTypes.STRING,
+				allowNull: false,
+				defaultValue: 'ACTIVE',
```

Comment:
@sumanvpacewisdom ACTIVE can be defined in common constant, so that you can access it from constant in all the places